editor/import/resource_importer_scene.cpp

```diff
@@ -101,25 +111,11 @@ Variant EditorSceneFormatImporter::get_option_visibility(const String &p_path, c
 	return ret;
 }
 
-//for documenters, these functions are useful when an importer calls an external conversion helper (like, fbx2gltf),
-//and you want to load the resulting file
-
-Node *EditorSceneFormatImporter::import_scene_from_other_importer(const String &p_path, uint32_t p_flags, int p_bake_fps) {
-	return ResourceImporterScene::get_singleton()->import_scene_from_other_importer(this, p_path, p_flags, p_bake_fps);
-}
-
-Ref<Animation> EditorSceneFormatImporter::import_animation_from_other_importer(const String &p_path, uint32_t p_flags, int p_bake_fps) {
-	return ResourceImporterScene::get_singleton()->import_animation_from_other_importer(this, p_path, p_flags, p_bake_fps);
-}
-
 void EditorSceneFormatImporter::_bind_methods() {
-	ClassDB::bind_method(D_METHOD("import_scene_from_other_importer", "path", "flags", "bake_fps"), &EditorSceneFormatImporter::import_scene_from_other_importer);
-	ClassDB::bind_method(D_METHOD("import_animation_from_other_importer", "path", "flags", "bake_fps"), &EditorSceneFormatImporter::import_animation_from_other_importer);
-
```

Comment:
why was this removed?

without this function, how can a user manually run the importer from GDScript?

Reply:
The original use-case of an importer calling an external conversion helper (like, fbx2gltf) doesn't make sense when you can call them directly. 

```gdscript
var gltf = EditorSceneFormatImporterGLTF.new() 
gltf._import_scene() 
var fbx = EditorSceneFormatImporterFBX.new() 
fbx._import_scene() 
```

import_scene_from_other_importer and import_animation_from_other_importer apis are removed.